fixtures/entries/procedure_performed.json

```diff
@@ -1,36 +1,62 @@
 {
-      "anatomical_approach": null,
-      "anatomical_target": null,
+  "anatomical_approach": null,
+  "anatomical_target": null,
+  "codes": {
+    "SNOMED-CT": [
+      "116783008"
+    ]
+  },
+  "description": "Procedure, Performed: Clinical Staging Procedure",
+  "end_time": 1426150800,
+  "incisionTime": 1428396000,
+  "method": null,
+  "mood_code": "EVN",
+  "negationInd": null,
+  "negationReason": null,
+  "oid": "2.16.840.1.113883.3.560.1.6",
+  "ordinality": null,
+  "performer_id": null,
+  "radiation_dose": null,
+  "radiation_duration": null,
+  "reaction": null,
+  "reason": null,
+  "source": null,
+  "specifics": null,
+  "start_time": 1426149000,
+  "status_code": {
+    "HL7 ActStatus": [
+      "performed"
+    ]
+  },
+  "time": null,
+  "values": [
+    {
       "codes": {
-        "ICD-10-PCS": [
-          "3E0F7DZ"
-        ],
         "SNOMED-CT": [
-          "24277005"
+          "369895002"
```

Comment:
What is the reason for the changes in this entry? Is it updates from a newer bundle?